app/commands/git/sync_concept_exercise.rb

```diff
@@ -51,6 +86,26 @@ def find_concepts(slugs)
       end
     end
 
+    def authors
+      head_git_exercise.authors.map do |author|
+        ::User.find_by!(handle: author["exercism_username"])
```

Comment:
Rather than having the n+1, you can do
```
::User.where(handle: head_git_exercise.authors.map{|a|a["exercism_username"]})
```

You could then compare the original list of usernames to the results of that query to get any that are missing for outputting.